README.md

```diff
@@ -21,90 +21,75 @@ Here are some of the benefits of using this library:
 **API Reference**: https://go.microsoft.com/fwlink/?linkid=2091700
 
 ### Feature Flags
-Feature flags are composed of two parts, a name and a list of feature-filters that are used to turn the feature on.
+Feature flags can either be on or off. They are composed of two parts, a name and a list of feature-filters that are used to turn the feature on.
 
 ### Feature Filters
-Feature filters define a scenario for when a feature should be enabled. When a feature is evaluated for whether it is on or off, its list of feature-filters are traversed until one of the filters decides the feature should be enabled. At this point the feature is considered enabled and traversal through the feature filters stops. If no feature filter indicates that the feature should be enabled, then it will be considered disabled.
+Feature filters define a scenario for when a feature flag should be enabled. When a feature flag is evaluated for whether it is on or off, its list of feature-filters are traversed until one of the filters decides the feature flag should be enabled. At this point the feature flag is considered enabled and traversal through the feature filters stops. If no feature filter indicates that the feature flag should be enabled, then it will be considered disabled.
 
 As an example, a Microsoft Edge browser feature filter could be designed. This feature filter would activate any features it is attached to as long as an HTTP request is coming from Microsoft Edge.
 
+### Dynamic Features
+Dynamic features can have values who's type range from object, to string, to integer and so on. Additionally, dynamic features can have an unlimited amount of values. A developer is free to choose what type should be returned when the value of a dynamic feature is requested. They are also free to choose how many options, known as variants, are available to select from.

```

Comment:
who's -> who have?

It feels a bit confusing to understand what we mean by a "value" of a dynamic feature. I think we can be more explicit that we are talking about the configuration of a dynamic feature under different conditions.